src/main/kotlin/com/terraformation/backend/accelerator/db/ReportStore.kt

```diff
@@ -1334,6 +1344,53 @@ class ReportStore(
           )
           .convertFrom { results -> results.map { ReportProjectMetricModel.of(it) } }
 
+  private fun userIsInSameOrg() =
+      with(ORGANIZATION_USERS) {
+        DSL.field(
+            if (currentUser() is SystemUser) DSL.falseCondition()
```

Comment:
Nit: Add curly braces to keep the two branches of the `if` consistently formatted.